lib/api/v1/user_api.rb

```diff
@@ -0,0 +1,200 @@
+module API
+  module V1
+    class UserAPI < Grape::API
```

Comment:
I think it's more aligned with rails if:

- The file is named `users.rb`.
- The class is named Users.

Plural because it's a resource, and without the `API` part because that's already stated by the name of the first module.